cpp/arcticdb/column_store/memory_segment_impl.hpp

```diff
@@ -240,7 +207,7 @@ class SegmentInMemoryImpl {
         }
 
         bool operator<(const Row &other) const {
-            return visit_field(0, [that=this, &other](auto type_desc_tag) {
+            return entity::visit_field(parent_->field(0), [that=this, &other](auto type_desc_tag) {
                 using RawType =  typename decltype(type_desc_tag)::DataTypeTag::raw_type;
                 return that->parent_->scalar_at<RawType>(that->row_id_, 0) <
                 other.parent_->scalar_at<RawType>(other.row_id_, 0);
```

Comment:
that=this is no longer necessary 
```suggestion
            return entity::visit_field(parent_->field(0), [this, &other](auto type_desc_tag) {
                using RawType =  typename decltype(type_desc_tag)::DataTypeTag::raw_type;
                return parent_->scalar_at<RawType>(that->row_id_, 0) <
                other.parent_->scalar_at<RawType>(other.row_id_, 0);
```